src/content/en/fundamentals/getting-started/primers/media-source-extensions.md

```diff
@@ -305,22 +311,22 @@ Source Extenstions. We recommend [stackoverflow](http://stackoverflow.com/questi
       console.log("The Media Source Extensions API is not supported.")
     }
     
-    
     function sourceOpen(e) {
       var mime = 'video/webm; codecs="opus, vp9"';
-      var mediaSource = this;
+      var mediaSource = e.target;
       var sourceBuffer = mediaSource.addSourceBuffer(mime);
       var videoUrl = 'droid.webm';
       fetch(videoUrl)
         .then(function(response) {
           return response.arrayBuffer();
         })
         .then(function(arrayBuffer) {
-          sourceBuffer.onupdateend = function(e) {
-            if (sourceBuffer.updating && mediaSource.readyState === 'open') {
+          sourceBuffer.addEventListener('updateend', function(e) {
+            if (!sourceBuffer.updating && mediaSource.readyState === 'open') {
               mediaSource.endOfStream();
+              // window.URL.revokeObjectURL(vidElement.src);
```

Comment:
1. Please remove `//` comment.
2. For consistency, let's use `URL.revokeObjectURL` instead of `window.URL.revokeObjectURL`